pumba/src/test/java/org/arquillian/cube/q/pumba/PumbaStandaloneContainerHandlerTest.java

```diff
@@ -0,0 +1,53 @@
+package org.arquillian.cube.q.pumba;
+
+import org.arquillian.cube.docker.impl.client.CubeDockerConfiguration;
+import org.arquillian.cube.q.spi.StandaloneContainer;
+import org.assertj.core.api.Assertions;
+import org.jboss.arquillian.core.api.Instance;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PumbaStandaloneContainerHandlerTest {
+
+    @Mock
+    CubeDockerConfiguration cubeDockerConfiguration;
+
+    @Test
+    public void shouldAddUnixVolume() {
+
+        Mockito.when(cubeDockerConfiguration.getDockerServerUri()).thenReturn("unix:///var");
```

Comment:
I like it more in such cases from the readability reasons. But again, it's more personal taste, so I leave it up to you. I